synapse/handlers/message.py

```diff
@@ -93,11 +93,11 @@ def __init__(self, hs):
 
     async def get_room_data(
         self,
-        user_id: str = None,
-        room_id: str = None,
-        event_type: Optional[str] = None,
-        state_key: str = "",
-        is_guest: bool = False,
+        user_id: str,
+        room_id: str,
+        event_type: str,
+        state_key: str,
+        is_guest: bool,
```

Comment:
This is called from one place, and it uses everything. The `event_type` isn't actually optional was what caused me to change these.

Reply:
OK FINE A COUPLE OF TESTS WERE USING IT BUT SHHHHHHH